posts/2021/08/low-code-workshop.md

```diff
@@ -0,0 +1,120 @@
+
+## Success stories
+
+Everyone loves an underdog story. 
+What follows are there triumphs of some open source allies who thought they could run an hour sprint that allowed multiple folks to contribute to an open source project. 
+Believe it or not, we weren't lying to ourselves and others. Below we highlight our successes with running low-code sprints in Jupyter and NumPy.
+
+### Jupyter accessibility group
+
+There is a small group of the Jupyter community focused on the improving the accessibility of JupyterLab and orbitting projects. 
+We meet every other work to add small patches and fixes. currently, we're in the process of organizing workshops to advocate for accessibility and improve the quality of the entire Jupyter experience for disabled scientists.
+
+On an off week, between our normal syncs, we tested a low-code format for folks to participate in the Jupyter project. 
+During an hour, four people included in the project by suggesting changes to alt text for images in the JupyterLab documentation (https://github.com/isabela-pf/jupyterlab/pull/1), 
+resulting in a collaborative commit to mainline with the help of @krassowski and @isabela-pf as champions for the shared work.(https://github.com/jupyterlab/jupyterlab/pull/10670)
+
+We consider this event a success because we were able to remove `git` as a technical barrier to entry and support changes from multiple authors.
+
+
+### [NumPy documentation](https://numpy.org/doc/stable/) new comers meeting
+
+Our fledgling experience in the Jupyter event made us *sort of* confident this could work for other projects. 
+So we contacted our NumPy accessibility allies @melissawm and @marsbarlee to test this
+hypothesis at the NumPy new comer's meeting, which [happens every other
+Thursday](http://numpy-discussion.10968.n7.nabble.com/Newcomer-s-Hour-td49226.html).
+
+During the meeting, @marsbarlee provided a valuable [slideshow on how to write alt-text for
+scientific diagrams](https://docs.google.com/presentation/d/150vhbpGrtAc3ALhrS1a07lhEKCgevAY3ITh-4eCndDk/edit?usp=sharing). 
+This involves looking at examples and looking at the image context.
+
+The progress from the NumPy new comer's meeting can be seen in this issue https://github.com/melissawm/numpy/pull/27 .
+
+A highlight, to us, is that we provided opportunities for both long time NumPy fans and high
+school students a chance to see themselves in history of such a foundational tool like NumPy. 
+Moreover, attendees were still adding alt text contributions after the event concluded.
+
+
+## Conclusion
+
+Using Github's suggested change system is a way to support low-code, single-line contributions to different open source code bases. 
+The event's collaborative hands-on format offers time for the community to interact in a meaningful way and see the fruits of their labor immediately.
+
+Please reach out to us if you or your project are interested in running event like this. 
```

Comment:
We might want to say and/or link to how to reach out. I don't know what the preferred way is, though. Is this direct to a community member listed above, or to Quansight Labs, or something else?

Reply:
maybe we can say to respond in the comments?

Reply:
I don't think we have commenting on the blog? Or am I not seeing it?

Reply:
There is commenting on the blog now. It does requires a github account. 
See https://github.com/Quansight-Labs/quansight-labs-site/pull/223 for background. 

Comments are actually reply backed by a GitHub issues – which is created once the first comment is posted.

Reply:
thanks for landing #233 that is rad!